<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addClientWidget</class>
 <widget class="QDialog" name="addClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="idLine">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cctvID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="cctvIP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipLine">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="registerClientBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
